src/modules/Common/components/Instances.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import Link from 'next/link';
+import Button from 'modules/Common/components/Button';
+import CardTable from 'modules/Common/components/CardTable';
+import CardTableItem from 'modules/Common/components/CardTableItem';
+import CardList from 'modules/Common/components/CardList';
+import LinkIcon from 'modules/Common/components/LinkIcon';
+import instancesData from '../../../../public/instances.json';
+import { kebabCase } from 'lodash';
+import { uniqueId } from 'lodash';
+import Card from 'modules/Common/components/Card';
+import { useTranslation } from 'next-i18next';
+import { useRouter } from 'next/router';
+
+type InstancesProps = {} & React.HTMLAttributes<HTMLDivElement>;
+
+const Instances: React.FC<InstancesProps> = ({ children, ...props }) => {
+  const { t } = useTranslation();
+  const router = useRouter();
+
+  // See https://github.com/microsoft/TypeScript/issues/41338
+  // @ts-ignore
+  const languageName = new Intl.DisplayNames(router.locale, { type: 'language' });
+  // @ts-ignore
+  const countryName = new Intl.DisplayNames(router.locale, { type: 'region' });
+
+  return (
+    <CardList title={t('instances:title')} centerTitle={true} big={true} {...props}>
+      {instancesData.instances.map((instance) => {
+        const slug = kebabCase(instance.name);
+        const author =
+          instance.maintainers.length == 0 ? (
+            <img src={`/images/contributors/volunteer-${router?.locale}.png`} />
```

Comment:
Not blocking

Here we should 
- have the same size for all maintainers and contributors assets
- use `next/image` with fixed size

Reply:
It's a pain to have the same image size for all logos or to have to declare their size in the `Instances.json` file to use `next/image`.

Let's talk about it.